tools/test-expect-version/test-expect-version.c

```diff
@@ -87,7 +90,7 @@ int main(int argc, char** argv)
     tty.c_cc[VTIME] = 5;
     tcsetattr(serialfd, TCSANOW, &tty);
 
-    alarm(60);
+    alarm(TIMEOUT);
```

Comment:
30 seconds may be too short in some cases to wait until the update is completed. 60 seconds was a safer choice to avoid false negatives on jenkins tests

Reply:
Set back to 60. accidental commit while testing.